HPCC-18755 Remove translated segMonitor support

richardkchapman opened the pull request:
Translated segmonitors are not serializable and cannot be executed remotely
(because of the associated generated code).

The value of 'offset' passed as the second parameter to createTranslatedKeySegmentMonitor
was random. This implies that the 'docompare' and 'copy' (used by index search
code will not correctly, work, though matchesBuffer (which is used when
using this SegMonitor for filtering) is ok.

Because they can't actually work, it's safest to remove them and greatly
simplifies refactoring.

VarOffset segmonitors similarly will not currently work in in-memory indexes
as the code for comparing rows is not implemented. Once refactoring of
segmonitors is completed this should become moot and they will no longer be
needed. Again, best to remove, to simplify the refactoring.

Signed-off-by: Richard Chapman <rchapman@hpccsystems.com>

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:
Manually observed a few cases.
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

richardkchapman committed:
HPCC-18755 Remove translated segMonitor support

Translated segmonitors are not serializable and cannot be executed remotely
(because of the associated generated code).

The value of 'offset' passed as the second parameter to createTranslatedKeySegmentMonitor
was random. This implies that the 'docompare' and 'copy' (used by index search
code will not correctly, work, though matchesBuffer (which is used when
using this SegMonitor for filtering) is ok.

Because they can't actually work, it's safest to remove them and greatly
simplifies refactoring.

VarOffset segmonitors similarly will not currently work in in-memory indexes
as the code for comparing rows is not implemented. Once refactoring of
segmonitors is completed this should become moot and they will no longer be
needed. Again, best to remove, to simplify the refactoring.

Distinction between docompare and docompareraw is now moot. All
implementations were identical, so simplify the code.

Never generate wild segmonitors - they can be added by the runtime.

Remove test that requires VarOffset segmonitors, as they are (temporarily, I
hope) unsupported.

Fixed some lurking bugs revealed by making finish() do some actual work.

Signed-off-by: Richard Chapman <rchapman@hpccsystems.com>

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 440cc1f37b3feca6565871c2e597f044ce2bd2e4
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |117 |117 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |743 |2 |
|test (thor) |640 |638 |2 |
|test (roxie) |773 |771 |2 |


[Errors:](https://gist.github.com/b9c0e42b9765823af784bbe08bce732d)
- test
  - hthor
    - 197\. [diskread (multiPart=false)](https://gist.github.com/HPCCSmoketest/b9c0e42b9765823af784bbe08bce732d#file-diskread-multipart-false-w20171117-064130-1-log), 198\. [diskread (multiPart=true)](https://gist.github.com/HPCCSmoketest/b9c0e42b9765823af784bbe08bce732d#file-diskread-multipart-true-w20171117-064130-9-log), 
  - thor
    - 177\. [diskread (multiPart=false)](https://gist.github.com/HPCCSmoketest/b9c0e42b9765823af784bbe08bce732d#file-diskread-multipart-false-w20171117-064454-5-log), 178\. [diskread (multiPart=true)](https://gist.github.com/HPCCSmoketest/b9c0e42b9765823af784bbe08bce732d#file-diskread-multipart-true-w20171117-064454-log), 
  - roxie
    - 197\. [diskread (multiPart=false)](https://gist.github.com/HPCCSmoketest/b9c0e42b9765823af784bbe08bce732d#file-diskread-multipart-false-w20171117-065142-5-log), 198\. [diskread (multiPart=true)](https://gist.github.com/HPCCSmoketest/b9c0e42b9765823af784bbe08bce732d#file-diskread-multipart-true-w20171117-065142-4-log), 

HPCC Stop: OK
HPCC Uninstall: OK

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 4bb2ccc72acf92f6526dba5ed8efaa06e0302ff3
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |117 |117 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |743 |2 |
|test (thor) |640 |638 |2 |
|test (roxie) |773 |771 |2 |


[Errors:](https://gist.github.com/8c6236ba7ae8ae53e5bf816eac10e415)
- test
  - hthor
    - 197\. [diskread (multiPart=false)](https://gist.github.com/HPCCSmoketest/8c6236ba7ae8ae53e5bf816eac10e415#file-diskread-multipart-false-w20171117-072949-3-log), 198\. [diskread (multiPart=true)](https://gist.github.com/HPCCSmoketest/8c6236ba7ae8ae53e5bf816eac10e415#file-diskread-multipart-true-w20171117-072950-log), 
  - thor
    - 177\. [diskread (multiPart=false)](https://gist.github.com/HPCCSmoketest/8c6236ba7ae8ae53e5bf816eac10e415#file-diskread-multipart-false-w20171117-073342-5-log), 178\. [diskread (multiPart=true)](https://gist.github.com/HPCCSmoketest/8c6236ba7ae8ae53e5bf816eac10e415#file-diskread-multipart-true-w20171117-073342-6-log), 
  - roxie
    - 197\. [diskread (multiPart=false)](https://gist.github.com/HPCCSmoketest/8c6236ba7ae8ae53e5bf816eac10e415#file-diskread-multipart-false-w20171117-074011-log), 198\. [diskread (multiPart=true)](https://gist.github.com/HPCCSmoketest/8c6236ba7ae8ae53e5bf816eac10e415#file-diskread-multipart-true-w20171117-074011-2-log), 

HPCC Stop: OK
HPCC Uninstall: OK

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: cf401b9c960c2a847be6c8b1047d1ed53f8dccae
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |116 |115 |0 |1 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Unittest error(s):
Run:  4   Failures: 0   Errors: 1

1) test: InMemoryIndexTest::test1 (E) 

uncaught exception of unknown type



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |631 |114 |
|test (thor) |640 |586 |54 |
|test (roxie) |773 |651 |122 |


[Errors:](https://gist.github.com/c9b0c6bf86d6d6a8ebea8890838d85da)
- test
  - hthor
    - 17\. [aggidx1 (multiPart=false, useSequential=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-aggidx1-multipart-false,-usesequential-true-w20171117-125557-10-log), 16\. [aggidx1 (multiPart=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-aggidx1-multipart-true-w20171117-125557-14-log), 18\. [aggidx2 (multiPart=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-aggidx2-multipart-true-w20171117-125557-11-log), 21\. [aggidx3 (multiPart=false, useSequential=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-aggidx3-multipart-false,-usesequential-true-w20171117-125558-log), 20\. [aggidx3 (multiPart=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-aggidx3-multipart-true-w20171117-125557-18-log), 23\. [aggidx4 (multiPart=false, useSequential=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-aggidx4-multipart-false,-usesequential-true-w20171117-125559-log), 22\. [aggidx4 (multiPart=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-aggidx4-multipart-true-w20171117-125558-2-log), 41\. [all_denormalize (multiPart=false)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-all_denormalize-multipart-false-w20171117-125603-3-log), 40\. [all_denormalize (multiPart=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-all_denormalize-multipart-true-w20171117-125603-log), 53\. [badindex](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-badindex-w20171117-125606-1-log), 67\. [bug5236 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-bug5236-multipart-true,-usetranslation-true-w20171117-125610-log), 87\. [childindex (multiPart=false)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-childindex-multipart-false-w20171117-125614-3-log), 88\. [childindex (multiPart=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-childindex-multipart-true-w20171117-125614-7-log), 90\. [childindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-childindex-multipart-true,-usetranslation-true-w20171117-125615-2-log), 112\. [countindex (multiPart=false)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-countindex-multipart-false-w20171117-125619-2-log), 113\. [countindex (multiPart=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-countindex-multipart-true-w20171117-125619-1-log), 114\. [countindex (multiPart=true, useLocal=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea889
 ... ( comment is too long, 50 lines, 50045 bytes, truncated) ... 
0c6bf86d6d6a8ebea8890838d85da#file-textsearch4-multipart-false-w20171117-130922-log), 719\. [textsearch4 (multiPart=true)](https://gist.github.com/HPCCSmoketest/c9b0c6bf86d6d6a8ebea8890838d85da#file-textsearch4-multipart-true-w20171117-130922-4-log), 

```

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: e3f0e64281cec825ac4f17a271e1c083a3b0c327
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |116 |115 |0 |1 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Unittest error(s):
Run:  4   Failures: 0   Errors: 1

1) test: InMemoryIndexTest::test1 (E) 

uncaught exception of unknown type



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |669 |76 |
|test (thor) |640 |627 |13 |
|test (roxie) |773 |692 |81 |


[Errors:](https://gist.github.com/302e2dad1ababf4e23f9d7d2689ce25f)
- test
  - hthor
    - 67\. [bug5236 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-bug5236-multipart-true,-usetranslation-true-w20171117-132458-3-log), 90\. [childindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-childindex-multipart-true,-usetranslation-true-w20171117-132503-1-log), 115\. [countindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-countindex-multipart-true,-usetranslation-true-w20171117-132508-log), 197\. [diskread (multiPart=false)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-diskread-multipart-false-w20171117-132526-9-log), 198\. [diskread (multiPart=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-diskread-multipart-true-w20171117-132526-3-log), 225\. [fetch (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-fetch-multipart-true,-usetranslation-true-w20171117-132532-3-log), 294\. [indexagg (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-indexagg-multipart-true,-usetranslation-true-w20171117-132552-2-log), 300\. [indexread (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-indexread-multipart-true,-usetranslation-true-w20171117-132553-1-log), 304\. [indexread2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-indexread2-multipart-true,-usetranslation-true-w20171117-132554-log), 316\. [indexread6 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-indexread6-multipart-true,-usetranslation-true-w20171117-132556-3-log), 359\. [keyed_denormalize (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-keyed_denormalize-multipart-true,-usetranslation-true-w20171117-132607-4-log), 366\. [keyed_join (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-keyed_join-multipart-true,-usetranslation-true-w20171117-132610-log), 370\. [keyed_join2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-keyed_join2-multipart-true,-usetranslation-true-w20171117-132610-3-log), 373\. [layouttrans (multiPart=true, version=1)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-layouttrans-multipart-true,-version-1-w20171117-132612-log), 374\. [layouttrans (multiPart=true, version=2)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d268
 ... ( comment is too long, 50 lines, 30332 bytes, truncated) ... 
dad1ababf4e23f9d7d2689ce25f#file-textsearch4-multipart-false-w20171117-133831-2-log), 719\. [textsearch4 (multiPart=true)](https://gist.github.com/HPCCSmoketest/302e2dad1ababf4e23f9d7d2689ce25f#file-textsearch4-multipart-true-w20171117-133831-5-log), 

```

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: fb1e4a2f9dcb3ae269019c448e2a62ec1c56a92b
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |116 |115 |0 |1 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Unittest error(s):
Run:  4   Failures: 0   Errors: 1

1) test: InMemoryIndexTest::test1 (E) 

uncaught exception of unknown type



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |723 |22 |
|test (thor) |640 |636 |4 |
|test (roxie) |773 |744 |29 |


[Errors:](https://gist.github.com/28bd8b28d6d253a3cd3de82eb7ebef81)
- test
  - hthor
    - 67\. [bug5236 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-bug5236-multipart-true,-usetranslation-true-w20171117-142234-4-log), 90\. [childindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-childindex-multipart-true,-usetranslation-true-w20171117-142239-2-log), 115\. [countindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-countindex-multipart-true,-usetranslation-true-w20171117-142244-3-log), 197\. [diskread (multiPart=false)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-diskread-multipart-false-w20171117-142303-1-log), 198\. [diskread (multiPart=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-diskread-multipart-true-w20171117-142303-4-log), 225\. [fetch (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-fetch-multipart-true,-usetranslation-true-w20171117-142309-5-log), 294\. [indexagg (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-indexagg-multipart-true,-usetranslation-true-w20171117-142328-3-log), 300\. [indexread (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-indexread-multipart-true,-usetranslation-true-w20171117-142330-4-log), 304\. [indexread2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-indexread2-multipart-true,-usetranslation-true-w20171117-142330-5-log), 316\. [indexread6 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-indexread6-multipart-true,-usetranslation-true-w20171117-142333-log), 359\. [keyed_denormalize (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-keyed_denormalize-multipart-true,-usetranslation-true-w20171117-142343-1-log), 366\. [keyed_join (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-keyed_join-multipart-true,-usetranslation-true-w20171117-142345-2-log), 370\. [keyed_join2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-keyed_join2-multipart-true,-usetranslation-true-w20171117-142346-4-log), 373\. [layouttrans (multiPart=true, version=1)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-layouttrans-multipart-true,-version-1-w20171117-142347-6-log), 374\. [layouttrans (multiPart=true, version=2)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de
 ... ( comment is too long, 50 lines, 11299 bytes, truncated) ... 
-true,-uselocal-true-w20171117-143617-log), 707\. [tablecount2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/28bd8b28d6d253a3cd3de82eb7ebef81#file-tablecount2-multipart-true,-usetranslation-true-w20171117-143617-5-log), 

```

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 06c48c2999c241b085fda53b16a1b237e49748ae
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |116 |115 |0 |1 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Unittest error(s):
Run:  4   Failures: 0   Errors: 1

1) test: InMemoryIndexTest::test1 (E) 

uncaught exception of unknown type



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |727 |18 |
|test (thor) |640 |616 |24 |
|test (roxie) |773 |755 |18 |


[Errors:](https://gist.github.com/824f0933b485cc64d2fec568cb0cdfd9)
- test
  - hthor
    - 67\. [bug5236 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-bug5236-multipart-true,-usetranslation-true-w20171117-145158-log), 90\. [childindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-childindex-multipart-true,-usetranslation-true-w20171117-145202-3-log), 115\. [countindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-countindex-multipart-true,-usetranslation-true-w20171117-145207-5-log), 197\. [diskread (multiPart=false)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-diskread-multipart-false-w20171117-145226-9-log), 198\. [diskread (multiPart=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-diskread-multipart-true-w20171117-145227-log), 225\. [fetch (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-fetch-multipart-true,-usetranslation-true-w20171117-145232-5-log), 294\. [indexagg (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-indexagg-multipart-true,-usetranslation-true-w20171117-145252-2-log), 300\. [indexread (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-indexread-multipart-true,-usetranslation-true-w20171117-145254-3-log), 304\. [indexread2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-indexread2-multipart-true,-usetranslation-true-w20171117-145255-1-log), 316\. [indexread6 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-indexread6-multipart-true,-usetranslation-true-w20171117-145257-4-log), 359\. [keyed_denormalize (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-keyed_denormalize-multipart-true,-usetranslation-true-w20171117-145307-4-log), 366\. [keyed_join (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-keyed_join-multipart-true,-usetranslation-true-w20171117-145309-1-log), 370\. [keyed_join2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-keyed_join2-multipart-true,-usetranslation-true-w20171117-145310-3-log), 373\. [layouttrans (multiPart=true, version=1)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-layouttrans-multipart-true,-version-1-w20171117-145312-log), 374\. [layouttrans (multiPart=true, version=2)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568
 ... ( comment is too long, 50 lines, 12271 bytes, truncated) ... 
setranslation-true-w20171117-151645-6-log), 707\. [tablecount2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/824f0933b485cc64d2fec568cb0cdfd9#file-tablecount2-multipart-true,-usetranslation-true-w20171117-151646-2-log), 

```

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 515377537b97f2343ae8f1c3d49d528668b3a00e
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |116 |115 |0 |1 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Unittest error(s):
Run:  4   Failures: 0   Errors: 1

1) test: InMemoryIndexTest::test1 (E) 

uncaught exception of unknown type



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |729 |16 |
|test (thor) |640 |618 |22 |
|test (roxie) |773 |757 |16 |


[Errors:](https://gist.github.com/2c5e96e33c0bb7c6f8e0788f488c4b9e)
- test
  - hthor
    - 67\. [bug5236 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-bug5236-multipart-true,-usetranslation-true-w20171120-052343-log), 90\. [childindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-childindex-multipart-true,-usetranslation-true-w20171120-052348-2-log), 115\. [countindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-countindex-multipart-true,-usetranslation-true-w20171120-052352-7-log), 225\. [fetch (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-fetch-multipart-true,-usetranslation-true-w20171120-052417-7-log), 294\. [indexagg (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-indexagg-multipart-true,-usetranslation-true-w20171120-052437-2-log), 300\. [indexread (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-indexread-multipart-true,-usetranslation-true-w20171120-052439-log), 304\. [indexread2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-indexread2-multipart-true,-usetranslation-true-w20171120-052439-2-log), 316\. [indexread6 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-indexread6-multipart-true,-usetranslation-true-w20171120-052442-5-log), 359\. [keyed_denormalize (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-keyed_denormalize-multipart-true,-usetranslation-true-w20171120-052453-1-log), 366\. [keyed_join (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-keyed_join-multipart-true,-usetranslation-true-w20171120-052455-2-log), 370\. [keyed_join2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-keyed_join2-multipart-true,-usetranslation-true-w20171120-052457-log), 373\. [layouttrans (multiPart=true, version=1)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-layouttrans-multipart-true,-version-1-w20171120-052457-3-log), 374\. [layouttrans (multiPart=true, version=2)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-layouttrans-multipart-true,-version-2-w20171120-052458-log), 375\. [layouttrans (multiPart=true, version=3)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-layouttrans-multipart-true,-version-3-w20171120-052459-log), 670\. [tablecount (multiPart=true, useTranslation=true)](htt
 ... ( comment is too long, 50 lines, 11296 bytes, truncated) ... 
setranslation-true-w20171120-054629-1-log), 707\. [tablecount2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/2c5e96e33c0bb7c6f8e0788f488c4b9e#file-tablecount2-multipart-true,-usetranslation-true-w20171120-054630-3-log), 

```

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: a687e471ea48d477d83516f646284b85f1a0a919
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |116 |115 |0 |1 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Unittest error(s):
Run:  4   Failures: 0   Errors: 1

1) test: InMemoryIndexTest::test1 (E) 

uncaught exception of unknown type



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |727 |18 |
|test (thor) |640 |618 |22 |
|test (roxie) |773 |755 |18 |


[Errors:](https://gist.github.com/88814e5499f6d618740bbb8a3ddbdd71)
- test
  - hthor
    - 63\. [bug12130 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-bug12130-multipart-true,-usetranslation-true-w20171120-060159-log), 67\. [bug5236 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-bug5236-multipart-true,-usetranslation-true-w20171120-060200-2-log), 90\. [childindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-childindex-multipart-true,-usetranslation-true-w20171120-060205-log), 115\. [countindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-countindex-multipart-true,-usetranslation-true-w20171120-060210-6-log), 225\. [fetch (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-fetch-multipart-true,-usetranslation-true-w20171120-060236-2-log), 294\. [indexagg (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-indexagg-multipart-true,-usetranslation-true-w20171120-060255-3-log), 300\. [indexread (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-indexread-multipart-true,-usetranslation-true-w20171120-060256-7-log), 304\. [indexread2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-indexread2-multipart-true,-usetranslation-true-w20171120-060257-5-log), 311\. [indexread4 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-indexread4-multipart-true,-usetranslation-true-w20171120-060259-log), 316\. [indexread6 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-indexread6-multipart-true,-usetranslation-true-w20171120-060259-1-log), 359\. [keyed_denormalize (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-keyed_denormalize-multipart-true,-usetranslation-true-w20171120-060310-6-log), 366\. [keyed_join (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-keyed_join-multipart-true,-usetranslation-true-w20171120-060312-5-log), 370\. [keyed_join2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-keyed_join2-multipart-true,-usetranslation-true-w20171120-060313-5-log), 373\. [layouttrans (multiPart=true, version=1)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-layouttrans-multipart-true,-version-1-w20171120-060314-3-log), 374\. [layouttrans (multi
 ... ( comment is too long, 50 lines, 12111 bytes, truncated) ... 
setranslation-true-w20171120-062704-3-log), 707\. [tablecount2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/88814e5499f6d618740bbb8a3ddbdd71#file-tablecount2-multipart-true,-usetranslation-true-w20171120-062705-4-log), 

```

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 9227b6cfc192cb7e8419f1b0243e943171e2829d
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |116 |115 |0 |1 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Unittest error(s):
Run:  4   Failures: 0   Errors: 1

1) test: InMemoryIndexTest::test1 (E) 

uncaught exception of unknown type



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |727 |18 |
|test (thor) |640 |638 |2 |
|test (roxie) |773 |755 |18 |


[Errors:](https://gist.github.com/e8ea870e94a6fe4986ab6bba809a527f)
- test
  - hthor
    - 63\. [bug12130 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-bug12130-multipart-true,-usetranslation-true-w20171120-064236-4-log), 67\. [bug5236 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-bug5236-multipart-true,-usetranslation-true-w20171120-064237-5-log), 90\. [childindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-childindex-multipart-true,-usetranslation-true-w20171120-064242-5-log), 115\. [countindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-countindex-multipart-true,-usetranslation-true-w20171120-064247-7-log), 225\. [fetch (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-fetch-multipart-true,-usetranslation-true-w20171120-064311-3-log), 294\. [indexagg (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-indexagg-multipart-true,-usetranslation-true-w20171120-064331-log), 300\. [indexread (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-indexread-multipart-true,-usetranslation-true-w20171120-064332-log), 304\. [indexread2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-indexread2-multipart-true,-usetranslation-true-w20171120-064333-log), 311\. [indexread4 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-indexread4-multipart-true,-usetranslation-true-w20171120-064334-2-log), 316\. [indexread6 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-indexread6-multipart-true,-usetranslation-true-w20171120-064335-4-log), 359\. [keyed_denormalize (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-keyed_denormalize-multipart-true,-usetranslation-true-w20171120-064345-10-log), 366\. [keyed_join (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-keyed_join-multipart-true,-usetranslation-true-w20171120-064348-log), 370\. [keyed_join2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-keyed_join2-multipart-true,-usetranslation-true-w20171120-064349-log), 373\. [layouttrans (multiPart=true, version=1)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-layouttrans-multipart-true,-version-1-w20171120-064350-log), 374\. [layouttrans (multiPart=t
 ... ( comment is too long, 50 lines, 8591 bytes, truncated) ... 
e,-usetranslation-true-w20171120-065610-log), 707\. [tablecount2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/e8ea870e94a6fe4986ab6bba809a527f#file-tablecount2-multipart-true,-usetranslation-true-w20171120-065611-log), 

```

ghalliday commented on ecl/hqlcpp/hqlsource.cpp:
```diff
@@ -4432,10 +4426,10 @@ void MonitorExtractor::buildKeySegmentExpr(BuildMonitorState & buildState, KeySe
                         createStringSet(subctx, targetSet, curSize, selectorInfo.selector->queryType());
                         OwnedHqlExpr targetVar = createVariable(targetSet, makeVoidType());
-                        callAddAll(subctx, targetVar);
+                        callAddAll(subctx, targetVar); // Not sure we need this really - library translates to wild anyway
```
I think needed because if it is used in combination with other conditional keyed filters

ghalliday commented on ecl/hqlcpp/hqlsource.cpp:
```diff
@@ -4432,10 +4426,10 @@ void MonitorExtractor::buildKeySegmentExpr(BuildMonitorState & buildState, KeySe
                     }
-                    else
-                        buildWildKeySegment(buildState, subctx, selectorInfo);
+//                    else
```
could delete

ghalliday commented on ecl/hqlcpp/hqlsource.cpp:
```diff
@@ -4665,20 +4578,20 @@ KeyedKind getKeyedKind(HqlCppTranslator & translator, KeyConditionArray & matche
 
 
-void MonitorExtractor::buildWildKeySegment(BuildMonitorState & buildState, BuildCtx & ctx, unsigned offset, unsigned size)
+void MonitorExtractor::buildWildKeySegment(BuildMonitorState & buildState, BuildCtx & ctx, unsigned fieldIdx, unsigned offset, unsigned size)
```
could remove

ghalliday commented on ecl/hqlcpp/hqlsource.cpp:
```diff
@@ -4901,24 +4802,8 @@ void MonitorExtractor::buildSegments(BuildCtx & ctx, const char * listName, bool
-            cur.generated = true;
-        }
-    }
+//    if (buildState.wildPending() && !ignoreUnkeyed)
```
delete?

ghalliday commented on rtl/eclrtl/eclhelper_dyn.cpp:
```diff
@@ -147,48 +147,55 @@ class FilterSet
+                irc->append(createWildKeySegmentMonitor(idx, offset, size)); // MORE - move this logic to irc::append ?
         }
+        // MORE - trailing wild needs adding
+        /*
```
is this needed or not?

richardkchapman replied:
Not any more

ghalliday commented on rtl/eclrtl/rtlkey.cpp:
```diff
@@ -24,186 +24,101 @@
     CKeySegmentMonitor(MemoryBuffer &mb)
     {
-        mb.read(size).read(offset).read(hash);
+        mb.read(size).read(offset).read(fieldIdx).read(hash);
```
Does this mean that the remote index code added in 6.4 cannot be used from 7.0?  (I'm not sure if that was the plan anyway).

ghalliday commented on rtl/eclrtl/rtlkey.cpp:
```diff
@@ -441,16 +308,17 @@ class CSingleKeySegmentMonitorBase : public CKeySegmentMonitor
     }
-    CSingleKeySegmentMonitorBase(bool _optional, unsigned _offset, const void *_val1, unsigned _val1size, const void *_val2, unsigned _val2size)
-        : CKeySegmentMonitor(_offset, _val1size+_val2size)
+    /*
```
delete?

ghalliday commented on rtl/eclrtl/rtlkey.cpp:
```diff
@@ -528,91 +390,46 @@ class CSingleKeySegmentMonitorBase : public CKeySegmentMonitor
         hash = FNV_32_HASHONE_VALUE(hash, (byte) 0);
         hash = FNV_32_HASHONE_VALUE(hash, (byte) 0);
     }
+    /*
```
delete?

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 71770fdc2df1dad5cb3469362d9a694a4382d0a0
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |116 |116 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |727 |18 |
|test (thor) |640 |638 |2 |
|test (roxie) |773 |755 |18 |


[Errors:](https://gist.github.com/ef123f0136974d08d619aab5520f39d0)
- test
  - hthor
    - 63\. [bug12130 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-bug12130-multipart-true,-usetranslation-true-w20171120-092830-2-log), 67\. [bug5236 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-bug5236-multipart-true,-usetranslation-true-w20171120-092831-4-log), 90\. [childindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-childindex-multipart-true,-usetranslation-true-w20171120-092836-2-log), 115\. [countindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-countindex-multipart-true,-usetranslation-true-w20171120-092841-3-log), 225\. [fetch (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-fetch-multipart-true,-usetranslation-true-w20171120-092907-log), 294\. [indexagg (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-indexagg-multipart-true,-usetranslation-true-w20171120-092926-log), 300\. [indexread (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-indexread-multipart-true,-usetranslation-true-w20171120-092927-2-log), 304\. [indexread2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-indexread2-multipart-true,-usetranslation-true-w20171120-092928-2-log), 311\. [indexread4 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-indexread4-multipart-true,-usetranslation-true-w20171120-092929-3-log), 316\. [indexread6 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-indexread6-multipart-true,-usetranslation-true-w20171120-092930-6-log), 359\. [keyed_denormalize (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-keyed_denormalize-multipart-true,-usetranslation-true-w20171120-092941-log), 366\. [keyed_join (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-keyed_join-multipart-true,-usetranslation-true-w20171120-092943-4-log), 370\. [keyed_join2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-keyed_join2-multipart-true,-usetranslation-true-w20171120-092944-3-log), 373\. [layouttrans (multiPart=true, version=1)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-layouttrans-multipart-true,-version-1-w20171120-092945-4-log), 374\. [layouttrans (multiPart=true, version=2)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-layouttrans-multipart-true,-version-2-w2017
 ... ( comment is too long, 41 lines, 8462 bytes, truncated) ... 
setranslation-true-w20171120-094233-2-log), 707\. [tablecount2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/ef123f0136974d08d619aab5520f39d0#file-tablecount2-multipart-true,-usetranslation-true-w20171120-094234-5-log), 

```

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 5fdfccea3a66d8655db8d517a7224dbcc62ae592
Empty table
Build: success
Build: success
ECL Watch:  Rebuilding Site
Empty table
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |116 |116 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |745 |727 |18 |
|test (thor) |640 |638 |2 |
|test (roxie) |773 |755 |18 |


[Errors:](https://gist.github.com/49ee968c8e7ac90e4423bb17d5df49d9)
- test
  - hthor
    - 63\. [bug12130 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-bug12130-multipart-true,-usetranslation-true-w20171120-102740-log), 67\. [bug5236 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-bug5236-multipart-true,-usetranslation-true-w20171120-102740-4-log), 90\. [childindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-childindex-multipart-true,-usetranslation-true-w20171120-102745-2-log), 115\. [countindex (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-countindex-multipart-true,-usetranslation-true-w20171120-102751-6-log), 225\. [fetch (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-fetch-multipart-true,-usetranslation-true-w20171120-102816-log), 294\. [indexagg (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-indexagg-multipart-true,-usetranslation-true-w20171120-102835-log), 300\. [indexread (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-indexread-multipart-true,-usetranslation-true-w20171120-102837-log), 304\. [indexread2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-indexread2-multipart-true,-usetranslation-true-w20171120-102837-4-log), 311\. [indexread4 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-indexread4-multipart-true,-usetranslation-true-w20171120-102839-3-log), 316\. [indexread6 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-indexread6-multipart-true,-usetranslation-true-w20171120-102840-5-log), 359\. [keyed_denormalize (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-keyed_denormalize-multipart-true,-usetranslation-true-w20171120-102851-log), 366\. [keyed_join (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-keyed_join-multipart-true,-usetranslation-true-w20171120-102853-5-log), 370\. [keyed_join2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-keyed_join2-multipart-true,-usetranslation-true-w20171120-102854-1-log), 373\. [layouttrans (multiPart=true, version=1)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-layouttrans-multipart-true,-version-1-w20171120-102855-5-log), 374\. [layouttrans (multiPart=true, version=2)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-layouttrans-multipart-true,-version-2-w20171120
 ... ( comment is too long, 41 lines, 8452 bytes, truncated) ... 
e,-usetranslation-true-w20171120-104110-log), 707\. [tablecount2 (multiPart=true, useTranslation=true)](https://gist.github.com/HPCCSmoketest/49ee968c8e7ac90e4423bb17d5df49d9#file-tablecount2-multipart-true,-usetranslation-true-w20171120-104111-log), 

```

richardkchapman committed:
HPCC-18755 Temporarily disable field translation tests

Disable tests of field translation - this is broken by this change, but will
be replaced soon and is not worth fixing.

Signed-off-by: Richard Chapman <rchapman@hpccsystems.com>
